### PR TITLE
Added CHANNEL variable in deploy_coreos_libvirt.sh

### PR DESCRIPTION
**Pull request by kayrus**
/cc @robszumski

### CODE REVIEW

File: os/deploy_coreos_libvirt.sh [hunk 1]
```diff
@@ -38,12 +39,12 @@ for SEQ in $(seq 1 $1); do
                 mkdir -p $LIBVIRT_PATH/$COREOS_HOSTNAME/openstack/latest || (echo "Can not create $LIBVIRT_PATH/$COREOS_HOSTNAME/openstack/latest directory" && exit 1)
         fi
 
-        if [ ! -f $LIBVIRT_PATH/coreos_production_qemu_image.img ]; then
-                wget http://stable.release.core-os.net/amd64-usr/current/coreos_production_qemu_image.img.bz2 -O - | bzcat > $LIBVIRT_PATH/coreos_production_qemu_image.img
+        if [ ! -f $LIBVIRT_PATH/coreos_${CHANNEL}_qemu_image.img ]; then
```
Comment (crawford) [inline os/deploy_coreos_libvirt.sh]: This isn't correct. The images are always called "coreos_production_*"

Comment (kayrus) [inline os/deploy_coreos_libvirt.sh]: It doesn't matter. I use pipe and "bzcat > $LIBVIRT_PATH/coreos_${CHANNEL}_qemu_image.img"

Comment (crawford) [inline os/deploy_coreos_libvirt.sh]: I see. Is there any reason not to use the standard names?

**Comment by kayrus** [comment]
you can run beta,alpha,stable without downloading new image every time

**Comment by marineam** [comment]
LGTM

**Pull request merged**